<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow {
    background-repeat: no-repeat;
    background-position: center;
}

QPushButton {
    background-color: transparent;
    border: none;
}</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QPushButton" name="SavedButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Level1Button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>500</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Level2Button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>520</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Level3Button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Level4Button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>560</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/resources/InstructPage.png</pixmap>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>SavedButton</zorder>
   <zorder>Level1Button</zorder>
   <zorder>PlayButton</zorder>
   <zorder>Level2Button</zorder>
   <zorder>Level3Button</zorder>
   <zorder>Level4Button</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
